binplace.targets

```diff
@@ -32,31 +32,24 @@
 
   <Target Name="GetTargetingPackDir">
     <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('NETCoreApp')) Or $(NuGetTargetMoniker.Contains('NETStandard'))">
-      <TargetingPackDir Include="$(BinDir)netcoreapp\TargetingPack" />
+      <TargetingPackDir Include="$(RefPath)" />
```

Comment:
I asked this on an earlier PR, but why do we still have this items set on a Target and not static?